modules/rosa-adding-taints.adoc

```diff
@@ -9,112 +9,121 @@
-                        <machine_pool_id>
-----
-<1> For machine pools that use autoscaling, you must provide minimum and maximum compute node replica limits. If you do not specify the arguments, you are prompted for the values before the command completes. The cluster autoscaler does not reduce or increase the machine pool node count beyond the limits that you specify. If you deployed ROSA using a single availability zone, the `--min-replicas` and `--max-replicas` arguments define the autoscaling limits in the machine pool for the zone. If you deployed your cluster using multiple availability zones, the arguments define the autoscaling limits in total across all zones and the counts must be multiples of 3.
-<2> Replace `<key>=<value>:<effect>,<key>=<value>:<effect>` with a key, value, and effect for each taint, for example `--taints=key1=value1:NoSchedule,key2=value2:NoExecute`. Available effects include `NoSchedule`, `PreferNoSchedule`, and `NoExecute`.This list overwrites any modifications made to node taints on an ongoing basis.
-+
-The following example adds taints to the `db-nodes-mp` machine pool:
-+
-[source,terminal]
-----
-$ rosa edit machinepool --cluster=mycluster --min-replicas=2 --max-replicas=3 --taints=key1=value1:NoSchedule,key2=value2:NoExecute db-nodes-mp
-----
-+
-.Example output
-[source,terminal]
-----
-I: Updated machine pool 'db-nodes-mp' on cluster 'mycluster'
-----
+// ifdef::openshift-rosa[]
+// . List the machine pools in the cluster:
+// +
+// [source,terminal]
+// ----
+// $ rosa list machinepools --cluster=<cluster_name>
+// ----
+// +
+// .Example output
+// +
+// [source,terminal]
+// ----
+// ID           AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS    TAINTS    AVAILABILITY ZONES    SPOT INSTANCES
+// Default      No           2         m5.xlarge                          us-east-1a            N/A
+// db-nodes-mp  No           2         m5.xlarge                          us-east-1a            No
+// ----
 
-.Verification
+// . Add or update the taints for a machine pool:
 
-. List the available machine pools in your cluster:
-+
-[source,terminal]
-----
-$ rosa list machinepools --cluster=<cluster_name>
-----
-+
-.Example output
-[source,terminal]
-----
-ID           AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS    TAINTS                                           AVAILABILITY ZONES    SPOT INSTANCES
-Default      No           2         m5.xlarge                                                                 us-east-1a            N/A
-db-nodes-mp  No           2         m5.xlarge                key1=value1:NoSchedule, key2=value2:NoExecute    us-east-1a            No
-----
-
-. Verify that the taints are included for your machine pool in the output.
-endif::[]
+// * To add or update taints for a machine pool that does not use autoscaling, run the following command:
+// +
+// [source,terminal]
+// ----
+// $ rosa edit machinepool --cluster=<cluster_name> \
```

Comment:
The same to above comments, replicas is not required for editing

Reply:
This section has been commented out.